src/napari_matplotlib/base.py

```diff
@@ -210,6 +210,9 @@ def current_z(self) -> int:
         Current z-step of the napari viewer.
         """
         return self.viewer.dims.current_step[0]
```

Comment:
```suggestion
```

Reply:
I assume the line `self.viewer.dims.current_step[-3]` is a bug fix below for datasets that have >= 4 dimensions?

Reply:
We can drop it for now since it does no affect this PR anymore. It had to do with other dimensions, yes